<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainGameWindow</class>
 <widget class="QMainWindow" name="MainGameWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>416</width>
    <height>325</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainGameWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>416</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>&amp;Game</string>
    </property>
    <addaction name="actionStart"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuSetting">
    <property name="title">
     <string>&amp;Setting</string>
    </property>
    <addaction name="actionBasic"/>
    <addaction name="actionMedium"/>
    <addaction name="actionHard"/>
    <addaction name="actionCustom"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menuSetting"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionStart">
   <property name="text">
    <string>&amp;New game</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="actionBasic">
   <property name="text">
    <string>&amp;Basic</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+B</string>
   </property>
  </action>
  <action name="actionMedium">
   <property name="text">
    <string>&amp;Medium</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+M</string>
   </property>
  </action>
  <action name="actionHard">
   <property name="text">
    <string>&amp;Hard</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+H</string>
   </property>
  </action>
  <action name="actionCustom">
   <property name="text">
    <string>&amp;Custom</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
